plugins/inputs/execd/README.md

```diff
@@ -34,6 +34,9 @@ STDERR from the process will be relayed to Telegraf as errors in the logs.
   ## Delay before the process is restarted after an unexpected termination
   restart_delay = "10s"
 
+  ## Timeout 
```

Comment:
Make a bit more descriptive. Something like `Timeout given to process when gracefully shutting down`